<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

    <ros2_control name="AGV" type="system">
        <hardware>
            <plugin>agv_hw/Tiger_motor</plugin>
            <param name="can_name">can0</param>
            <param name="motor_id_left">2</param>
            <param name="motor_id_right">1</param>
        </hardware>
        <joint name="left_wheel_joint">
            <command_interface name="velocity"/>
            <state_interface name="position"/>
        </joint>
        <joint name="right_wheel_joint">
            <command_interface name="velocity"/>
            <state_interface name="position"/>
        </joint>
    </ros2_control>

</robot>